bin/rails

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env ruby
-APP_PATH = File.expand_path('../config/application', __dir__)
+APP_PATH = File.expand_path("../config/application", __dir__)
```

Comment:
Looks like automated update to latest upstream code.

Reply:
Yes, this is how the autogenerated code looks like in the new version...

Running `bundle exec rails app:update` overwrites (well, only if you want to) all of the Rails default files. While I could have skipped all the style changes, I decided to keep them, so that it's easier on posterior upgrades... 
Apparently, now Rails developers prefer double quotes :shrug: